src/main.js

```diff
@@ -492,9 +492,8 @@ const CodingChatboxMainWindow = new Lang.Class({
                 if (!row)
                     throw new Error('Couldn\'t find row matching actor ' + actor);
 
-                // TODO: Translations
-                this.application.showNotification('Waiting on your input',
-                                                  actor + ' is still waiting on your response!',
+                this.application.showNotification(_('Waiting on your input'),
```

Comment:
We usually use double quotes for translatable strings. (multiple occurrences)
I think that in the past we did it because something was not working with single quotes, but maybe that's not an issue anymore? If things seem to work fine, I'd be inclined to keep it like this.